src/app/twinclass/[twinClassId]/views/twin-class-general.tsx

```diff
@@ -283,6 +286,18 @@ export function TwinClassGeneral() {
             </TableCell>
           </TableRow>
 
+          <TableRow>
+            <TableCell>View Permission</TableCell>
+            <TableCell>
+              {(twinClass.viewPermission && (
+                <PermissionResourceLink
+                  data={twinClass.viewPermission as PermissionV1}
+                  withTooltip
+                />
+              )) || <NoneDataText />}
```

Comment:
1. Let's remove `<NoneDataText />` by deleting `/twinfaces/src/components/NoneDataText.tsx` file
2. Update `<TableCell>`  to something like
```ts
const TableCell = React.forwardRef<
  HTMLTableCellElement,
  React.TdHTMLAttributes<HTMLTableCellElement>
>(({ className, ...props }, ref) => {
  if (isUndefined(props.children)) {
    return <div className="text-gray-700 font-light">None</div>;
  }

  return (
    <td
      ref={ref}
      className={cn(
        "p-4 align-middle [&:has([role=checkbox])]:pr-0",
        className
      )}
      {...props}
    />
  );
});
```